flask_caching/backends/__init__.py

```diff
@@ -25,6 +25,7 @@
 )
 from flask_caching.backends.simplecache import SimpleCache
 from flask_caching.backends.uwsgicache import UWSGICache
+from flask_caching.contrib.ignitecache import IgniteCache
```

Comment:
Don't include your backend here. This is only for officially supported backends and not for user contributed backends.